### PR TITLE
time to remove skip_english_translation, not in globals anymore

### PR DESCRIPTION
**Pull request by stephenwaite**
discovered from [post on forum](https://community.open-emr.org/t/fatal-error-in-opening-patient-document-template-forms-in-newly-upgraded-5-0-2-1/14309/11?u=stephenwaite)

### CODE REVIEW

**Comment by bradymiller** [comment]
note there's several more places where need to remove this (why is it things in OpenEMR are never simple :) )

image

**Comment by stephenwaite** [comment]
right on, jeez, how did i miss those

File: interface/language/lang_definition.php [hunk 1]
```diff
@@ -42,20 +42,24 @@
 
 ?>
 
-<form name='filterform' id='filterform' method='post' action='?m=definition&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>' onsubmit="return top.restoreSession()">
+<form name='filterform' id='filterform' method='post'
+      action='?m=definition&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>'
+      onsubmit="return top.restoreSession()">
```
Comment (bradymiller) [inline interface/language/lang_definition.php:47]: In future, rec avoiding changes like this. For example, it appears @tywrenn 's editor will do the opposite of this and revert this change if he edits this script. Thus starting a infinite war of code changes :)

**Comment by bradymiller** [comment]
looks good

File: interface/language/lang_definition.php (hunk 1 above)
Comment (stephenwaite) [reply interface/language/lang_definition.php:47]: ok, it's actually a `PSR12` warning, like the below which is going to be a bear if it progresses to error stage
```
  56 | WARNING | Line exceeds 120 characters; contains 136 characters
```

**Pull request merged**